recipes/jerryscript/all/conanfile.py

```diff
@@ -14,15 +16,82 @@ class JerryScriptStackConan(ConanFile):
     options = {
         "shared": [True, False],
         "fPIC": [True, False],
+        "tool_cmdline": [True, False],
+        "tool_cmdline_test": [True, False],
+        "tool_cmdline_snapshot": [True, False],
+        "libfuzzer_support": [True, False],
```

Comment:
Is libfuzzer support of interest to users?
Mostly, fuzzer/sanitizer options are there for developers to debug/find problems.

Reply:
Again, because it's a script engine, I thought it might be used for something the end user might benefit from. I looked a little deeper and found that you were right in questioning me here. Will remove.

Reply:
Thanks, and thank you for taking the time to improve this recipe.

I am also reviewing to make sure maintaining this recipe remains tenable.